src/NServiceBus.RabbitMQ/Configuration/ConnectionConfiguration.cs

```diff
@@ -31,7 +31,7 @@ class ConnectionConfiguration
 
         public string CertPassphrase { get; set; }
 
-        public IDictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
+        public Dictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
```

Comment:
What is the reason for this change?

We gain the specificity of the type, but we lose the ability to change the concrete type.